driver/src/main/java/oracle/nosql/driver/ops/ReplicaStatsResult.java

```diff
@@ -0,0 +1,151 @@
+/*-
+ * Copyright (c) 2011, 2023 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at
+ *  https://oss.oracle.com/licenses/upl/
+ */
+
+package oracle.nosql.driver.ops;
+
+import java.util.Collections;
+import java.util.Map;
+
+import oracle.nosql.driver.NoSQLHandle;
+import oracle.nosql.driver.values.TimestampValue;
+
+/**
+ * Cloud service only.
+ * <p>
+ * ReplicaStatsResult is returned from {@link NoSQLHandle#getReplicaStats}.
+ * It contains replica statistics for the requested table.
+ * @see NoSQLHandle#getReplicaStats
+ *
+ * @since 5.4.13
+ */
+public class ReplicaStatsResult extends Result {
+    private String tableName;
+    private long nextStartTime;
+    private Map<String, ReplicaStats[]> statsRecords;
+
+    /**
+     * Returns the table name used by the operation
+     *
+     * @return the table name
+     */
+    public String getTableName() {
+        return tableName;
+    }
+
+    /**
+     * Returns the next start time. This can be provided to
+     * {@link ReplicaStatsRequest} to be used as a starting point or listing
+     * replica stats records.
+     *
+     * @return the next start time
+     * @since 5.4
```

Comment:
don't need since here, it's in the main doc for this class

Reply:
Fixed.